src/main/webapp/search-meal-style.css

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2020 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License. */
+
+body {
+  background-image: url("image/background-main.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: right top;
+  background-attachment: fixed;
+  font-family: sans-serif;
```

Comment:
Technically all these statements can be put into the `style.css` file, and the page-specific CSS files would only specify the `background-image`, and on the page will be a style that is the combination of both.

However I don't know if that's good practice for two reasons: I don't know if it's a good practice to merge multiple styles by means other than classes (i.e. having `body`, `body.meal` and `body.search` styles), and I don't know if separating `background-image` from the rest of the "background" parameters is a good idea.

I need Tomasz' feedback on this.

Reply:
Technically speaking, all selectors can be moved to a single .css file (and that's going to be our chosen approach when using a framework). We can delay fully merging all those files into one until later on, because having multiple css files is an acceptable approach, too.

I wouldn't worry too much about having `background-` properties set in different selectors. Consistency is key in UI, so having a property defined once is usually a good idea.

I'd suggest:
1. moving the common bits to style.css with a `body` selector
2. either
  - leaving the remaining bits (in this case `background-image` in the specific file, as it is now)
  - or moving them to the main file as well, but adding an id (e.g. `#search-meal-body`, `#meal-page-body`) or class selector (e.g. `.with-background-meal`, `.with-background-main`) that would only define the specific fields (here: `background-image`).

I can see there's also a margin-right property applied to `body`. It's usually a good idea to define shape only on more specific elements, e.g. `div`.